webapp/channels/src/utils/text_formatting.tsx

```diff
@@ -618,6 +637,10 @@ export function escapeRegex(text?: string): string {
     return text?.replace(/[-/\\^$*+?.()|[\]{}]/g, '\\$&') || '';
 }
 
+export function escapeReplaceEspecialPatterns(text?: string): string {
+    return text?.replace(/[$]/g, '$$$$') || '';
```

Comment:
Duh, this is a replace pattern too, that's why 😄

Reply:
Actually, just three dollar signs would result in the same, since the last one will not be considered part of a special pattern. But I feel 4 makes more sense :sweat_smile: